src/pages/users/[userid].tsx

```diff
@@ -0,0 +1,153 @@
+import type { GetServerSideProps, InferGetServerSidePropsType, NextPage } from 'next';
+import { Navbar } from '../../components/Navbar';
+import { useSession } from 'next-auth/react';
+import { TourSiteCard } from '../../components/TourSiteCard';
+import Router, { useRouter } from 'next/router';
+import { Tour, User } from '@prisma/client';
+import { getToken } from 'next-auth/jwt';
+import { prisma } from "../../lib/prisma";
+import { CreateTourResponseType } from '../api/tour';
+import { useEffect, useState } from 'react';
+import Error from 'next/error';
+
+
+const NewPage: NextPage = ({ propTours, userid }: InferGetServerSidePropsType<typeof getServerSideProps>) => {
+    const router = useRouter();
+    const { data: session, status } = useSession();
+    const [tours, setTours] = useState(propTours);
+    const [query, setQuery] = useState("");
+    const [changing, setChanging] = useState(false);
+    let timer: NodeJS.Timeout;
+  
+    const queryTours = () => {
```

Comment:
Eventually, we will make this card search functionality its own component, but this is good for now.